src/core/speechrecognition/support.js

```diff
@@ -9,5 +9,5 @@
  * @returns {boolean}
  */
 export function speechRecognitionIsSupported () {
-  return !!(window.SpeechRecognition || window.webkitSpeechRecognition);
+  return !!(window.webkitSpeechRecognition && navigator.mediaDevices);
```

Comment:
On the browsers that we support (Edge, Chrome and Safari) the api is available at window.webkitSpeechRecognition and not at window.SpeechRecognition, so I think we're fine to remove that. @oshi97 Are there any reasons that you can think of for leaving it in?

Reply:
I think in the future we might need to check for SpeechRecognition since that's the official API spec, so I would probably leave it in. On the other hand, if something changes about the API when it moves from webkitSpeechRecognition to just SpeechRecognition we will need to make changes anyways. I'm fine either way

Reply:
I'd prefer that we manually verify that the official `SpeechRecognition` API works the same rather than having us automatically start using that API once it's available so I think we're good to leave as-is